swagger2.0/oic.sec.doxm.swagger.json

```diff
@@ -142,7 +140,9 @@
         "if": {
           "description": "The interface set supported by this Resource.",
           "items": {
-            "enum": [ "oic.if.baseline", "oic.if.rw" ],
+            "enum": [
+              "oic.if.baseline"
```

Comment:
Why has this gone from supporting ["oic.if.baseline" , "oic.if.rw"]  to ["oic.if.rw"] only?

Reply:
Thought I'd caught all of those; Oleks' changes were missing the "oic.if.rw" change, so I added that in, must have missed this one.  I've pushed a new commit that fixes it.